test/services/trails_service_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class TrailsServiceTest < ActiveSupport::TestCase
+  attr_reader :service
+
+  def setup
+    @service = TrailsService.new
+  end
+
+  test '#trails' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.